### PR TITLE
Bubble Sort Algorithm

### PR DESCRIPTION
**Pull request by Samitti**
We did two assignments from Odin projects on the bubble sort algorithm.

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
The project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your did a great job!
Just a few minor changes left:
* [ ]  README file should have a short description on how to run your project.
* [ ] There should be no debugging instructions in committed code (as discussed [here](https://questions.microverse.org/t/debugging-instructions-code/346)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/section/debugging_requirement.png) from Pathwright)
* [ ] Rename `sort_bubble` method to `bubble_sort_by`, so it's easier to understand what that method does.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by NiiazalyDzhumaliev** [comment]
Thanks for the review. We changed the name of bubble_sort to bubble_sort_by, add a description to the README file and in the future won't use debugging instructions in commits. Please, kindly review our project.

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    if arr[i] > arr[i + 1]
+      arr[i], arr[i + 1] = arr[i + 1], arr[i]
+      i = 0
+    else
+      i += 1
+    end
+  end
+  arr
+end
+puts bubble_sort([4, 3, 78, 2, 0, 2])
```
Comment (stratospherique) [inline bubble_sort.rb]: - There should be no debugging instructions in committed code.

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+def sort_bubble_by(array)
+  array.each do
+    (array.length - 1).times do |num|
+      array[num], array[num + 1] = array[num + 1], array[num] if yield(array[num], array[num + 1]).positive?
+    end
+  end
+  array
+end
+puts sort_bubble(%w[hello hi hey]) { |right, left| right.length - left.length }
```
Comment (stratospherique) [inline bubble_sort.rb]: - There should be no debugging instructions in committed code.

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: Congrats, the project is approved :heavy_check_mark: You may merge :+1: :+1: :+1: 

Keep up the good work :muscle: 
**_Ahmed Mahfoudh_**
#### PS: Check the code comments below, they indicate a minor issue in your code that you should fix before you merge.

**Pull request merged**